src/app/detail/ui/styledComponent.tsx

```diff
@@ -81,4 +81,27 @@ export const TapWriteComment = styled(Tapwrite)(({ theme }) => ({
 
 export const TapWriteTaskEditor = styled(Tapwrite)(({ theme }) => ({
   minHeight: '30vh',
+  fontSize: '20px',
 }))
+
+export const TapWriteTaskEditor1 = styled(Tapwrite)(({ theme }) => ({
+  minHeight: '20vh',
+  borderColor: theme.color.borders.border,
+  border: `1px solid ${theme.color.base.black}`,
+}))
+
+export const StyledContainer = styled('div')<{ padding?: string; borderColor?: string; borderLess?: boolean }>(
+  ({ padding, borderColor, borderLess, theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+
+    padding: padding ? padding : '3px 8px 3px 0px',
+    borderColor: borderColor ? borderColor : theme.color.borders.border,
```

Comment:
```suggestion
    padding: padding ?? '3px 8px 3px 0px',
    borderColor: borderColor ?? theme.color.borders.border,
```

Reply:
removed styledContainer because it was unused. Instead of using styledContainer, made some changes in tapwrite and forgot to delete this component. Updated in the latest push